src/Nordea/Components/MultiSelectDropdown.elm

```diff
@@ -160,37 +178,48 @@ view attrs dropdown =
                         , maxHeight (rem 16.75)
                         , dropdownStyles
                         , listStyle none
+                        , Css.property "margin-block-start" "0"
+                        , Css.property "margin-block-end" "0"
+                        , Css.property "padding-inline-start" "0"
                         ]
                     ]
                     (dropdown.options |> List.map viewOption)
                 ]
     in
     Html.fieldset
         ([ Events.on "focusout" (Decode.succeed (dropdown.onFocus False))
-         , Events.on "focusin" (Decode.succeed (dropdown.onFocus True))
+         , Events.onClick (dropdown.onFocus (not dropdown.hasFocus))
          , tabindex 0
-         , css [ displayFlex, flexDirection column ]
+         , css
+            [ minWidth fitContent
+            , zIndex (int 1)
+            , displayFlex
+            , flexDirection column
+            , marginInlineStart (rem 0)
+            , marginInlineEnd (rem 0)
+            , Css.property "padding-block-start" "0"
+            , Css.property "padding-block-end" "0"
+            , Css.property "padding-inline-start" "0"
+            , Css.property "padding-inline-end" "0"
```

Comment:
I think this should be enough; setting the shorthand should set all sub-variants too
```suggestion
            , margin (rem 0) 
            , padding (rem 0)
```

Reply:
Also needed to set this on `ul`, but other than that it worked with setting margin and padding to 0🙏